src/v2/cookbook/creating-custom-scroll-directives.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Creating Custom Scroll Directives
+type: cookbook
+order: 5
+---
+
+## Simple Example
+
+There are many times that we might want to add a bit of behavior, especially animation, to a scroll event on a site. There are many ways to do so, but the path with the least amount of code and dependencies is to use a [custom directive](https://vuejs.org/v2/guide/custom-directive.html) to create a hook for anything that fires off a particular scroll event.
```

Comment:
Maybe `is perhaps to use…` instead? Unless we're 100% sure that a directive is _always_ shorter.

Reply:
Ah good point. I can't think of a time when it wouldn't be shorter but I'm sure there are edge cases out there :)